.github/pull_request_template.md

```diff
@@ -1,51 +1,40 @@
-## What does this change?
-
 <!--
-**NOTE**: This is meant as a guide and memory jog, not a prescriptive list of tasks you must do!
+This is meant as a guide and memory jog, not a prescriptive list of tasks you must do!
 Feel free to modify as suits, just please do read over and make sure you've at least thought about the points and if there's any impact.
 -->
 
 <!--
-Add details of the change here. A PR should have enough detail to be understandable far in the future. e.g what is the problem/why is the change needed, how does it solve it and any questions or points of discussion.
+Append the ticket *task ID* to the PR description like so: Awesome update [GEN-123]
 
-Add any relevant tickets or links, eg: Resolves [ticket](http://link-to-ticket)
+Notion will detect this and add a link to the original ticket as a comment.
 -->
 
 ## Testing
 
-- [ ] I wrote new unit tests for this change
-- [ ] I wrote new integration/e2e tests for this change
-- [ ] I updated existing tests for this change
-- [ ] I ran test commands to test this change (list below)
```

Comment:
I personally didn't find these useful as I think adding tests is too basic to warrant a specific reminder and tests should always be run as part of CI anyway so no need to specify which ones have been run (they're all going to be run).

Reply:
> adding tests is too basic to warrant a specific reminder
Sometimes they still get missed though! I think we could merge the unit/e2e to "added tests if required" but would rather keep a little reminder in
> tests should always be run as part of CI anyway 
That prompt is a reminder to run locally before opening the PR. I often find it a good reminder (saves our credits)